Build/linq2db.Providers.props

```diff
@@ -0,0 +1,78 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I moved these over because I couldn't get the new tests to build. The tests reference these libraries statically and won't build if they are not referenced. This file only applies to the testing projects anyway.

Reply:
I had removed all the other provider specific tests so those libs weren't referenced and it worked. I will take a look.

Reply:
Sounds good. Maybe you can show me what you did when we meet next. I am perfectly happy to remove as many dependencies as we can.